feconf.py

```diff
@@ -641,6 +641,8 @@ def get_empty_ratings():
 
 # Bool to enable update of analytics models.
 ENABLE_NEW_STATS_FRAMEWORK = False
+# Current event models schema version.
```

Comment:
Maybe document past schema versions here, as well as the reasons why the schema version changed.

An alternative is to have a BaseEventModel in core.storage.statistics.gae_models and have all other event models inherit from it; that way you can keep this constant in that class and also do the put() overwriting in a single place there. That might be cleaner, actually.

Reply:
@seanlip The BaseEventModel actually sounds like a better idea. But then, we still don't have a lot of reused variables like the schema version. If we need to make a base model just for the schema version, wouldn't it make sense to just have it in feconf?

Reply:
I was thinking about the put(), too. If it's just the schema version, I'm not particularly bothered; if it's both that and the put() then it starts to make more sense to have a BaseEventModel.

Reply:
Though now based on my other comment we probably don't need BaseEventModel, at least not yet, so you can ignore this comment too.